autoload/lsp.vim

```diff
@@ -4,6 +4,8 @@ let s:servers = {} " { lsp_id, server_info, init_callbacks, init_result, buffers
 
 let s:notification_callbacks = [] " { name, callback }
 
+let s:file_content = {}
```

Comment:
Because vim is not statically typed languages. I prefer to document the structure of what `file_content` looks like with comments. or if it certain things are optional. Makes it easy in the future when we are reading code.